templates/index.md.tmpl

```diff
@@ -39,17 +43,27 @@ URL                        EMAIL                                         USER ID
 acme.dev.com               test.user@acme.com                            HERE_WILL_BE_REAL_USER_ID
 acme.prod.com              prod.user@acme.com                            HERE_WILL_BE_REAL_USER_ID
 ```
-3. Set `account` in the provider configuration with the `URL` or `USER ID` value from the previous step.
+3. Set `OP_ACCOUNT` environment variable or `account` in the provider configuration with the `URL` or `USER ID` value from the previous step.
 4. When the biometric unlock popup appears while running terraform command, [authenticate it using fingerprint or password](https://developer.1password.com/docs/cli/app-integration/#step-2-enter-any-command-to-sign-in).
 
 ## Use with 1Password Connect
 
 To use the provider with 1Password Connect you need to
 1. [Deploy your Connect server](https://developer.1password.com/docs/connect/get-started#deployment)
-2. Set `url` and `token` in the provider configuration.
+2. Set `OP_CONNECT_HOST` environment variable or `url` in the provider configuration.
+3. Set `OP_CONNECT_TOKEN` environment variable or `token` in the provider configuration.
 
 ## Example Usage
 
 {{tffile "examples/provider/provider.tf"}}
 
+Each of the configuration options can be set using env variables.
+That's more secure way to do that as prevents from accidentally pushing sensitive values to the project repo.
```

Comment:
`[nitpick]`
Better English phrasing:
```suggestion
Each of the configuration options can be set using env variables.
This is a more secure way to do provide your credentials to avoid accidentally pushing sensitive values to the project repository.
```